### ISSUE TITLE
Completion codes

### ISSUE THREAD

**Issue by Shagun-n** [ashiksm989/EveryDayExcel#1]
Completion codes are missing in all your assignments. Where can I find those?

**Comment by ashiksm989** [comment]
Give me some time i will provide it to you

**Comment by Shagun-n** [comment]
Can you please provide them this week?
My course is about to expire within a few days and I'm unable to solve the assignments.

**Comment by ashiksm989** [comment]
Everyday Excel Part 3
Amortization Schedule with extra payment 2876
Bakery Shopping list 4561
dinner sign up 8454
Dynamic amortization schedule 1939
Dynamic Data Cleaning 6267
Dynamic Temperature Lookup 4780
Friday the 13th 3489
Historical Weathe lookup  Part A 4190
Longest Ladder around the corner 3377
Nearest Eighth of an inch 9381
Real time regression 1903
Sensitivity analysis 6093
world bank lookup 5271
Rental Center Inventory 5298
Distance Calculator 4949


Everyday Excel Part 2
Week 1 
Assignment 1 7003
Week 2
Assignment 2 5397
Week 3
Assignment 3.1 7027
Assignment 3.2 6237
Week 4
Assignment 4 3838
Week 5
Assignment 5 1994

Everyday Excel Part 1
Week 1
Assignment 1 3027
Week 2
Assignment 2 7390
Week 3
Assignment 3.1 1279
Assignment 3.2 1279
Week 4
Assignment 4.1 3871
Assignment 4.2 5734
Week 5
Assignment 5.1 7301
Assignment 5.2 2998

**Comment by Shagun-n** [comment]
Can you also provide submission codes for:
a) Distance Calculator(Week 4)
b) Rental Center Inventory(Week 4)
c) Historical Weather Lookup PART B(Week 4)
d) Both submissions of honors assignment( Week 5)

Rest all are done.

**Comment by ashiksm989** [comment]
I can't finished Historical Weather Lookup and honors assignment. But i have excel file of Distance calculator and rental center inventory

**Comment by ashiksm989** [comment]
Distance Calculator code is 4949. Rental Center inventory code 5298

**Comment by ashiksm989** [comment]
If you complete the those. can you provide me the excel file. actually i didn't give much effort to complete the those.

**Comment by omm003** [comment]
someone provide me the completion code of  Historical Weather Lookup (Part B) please its urgent....

**Comment by omm003** [comment]
does anyone have the completion code of Francesca's French Bakery

Top Hat Consolidator

in honours project of everyday excel

**Comment by nandz123827** [comment]
> someone provide me the completion code of Historical Weather Lookup (Part B) please its urgent....

Yes i need that also bro please completion code of Historical Weather Lookup (Part B) please
